tests/Resources/ti.locale.test.js

```diff
@@ -0,0 +1,290 @@
+		should(Ti.Locale.getLocaleCurrencySymbol('en-US')).eql('$');
+	});
+
+	// FIXME Get working on iOS
+	// FIXME Get working properly cross-platform. JPY gives us ¥ on Windows and Android, JP¥ on iOS. CNY gives us ¥ on Windows, CN¥ on Android
+	it.androidAndIosBroken('#getCurrencySymbol(String)', function () {
+		should(Ti.Locale.getCurrencySymbol).be.a.Function();
+		should(Ti.Locale.getCurrencySymbol('USD')).eql('$');
+		should(Ti.Locale.getCurrencySymbol('JPY')).eql('¥'); // 'JP¥' on iOS
+		should(Ti.Locale.getCurrencySymbol('CNY')).eql('¥'); // 'CN¥' on Android
+		should(Ti.Locale.getCurrencySymbol('TWD')).eql('NT$');
+	});
+
+	it('#getCurrencyCode(String)', function () {
+		should(Ti.Locale.getCurrencyCode).be.a.Function();
+		should(Ti.Locale.getCurrencyCode('en-US')).eql('USD');
+		should(Ti.Locale.getCurrencyCode('ja-JP')).eql('JPY');
+		should(Ti.Locale.getCurrencyCode('zh-CN')).eql('CNY');
+		should(Ti.Locale.getCurrencyCode('zh-TW')).eql('TWD');
+	});
+
+	// TODO Support Ti.Locale.formatTelephoneNumber on other platforms?
+	it.android('#formatTelephoneNumber(String)', function () {
+		should(Ti.Locale.formatTelephoneNumber).be.a.Function();
+		// TODO Actually check inputs/outputs!
+	});
+
+	it('currentCountry', function () {
+		should(Ti.Locale.currentCountry).be.a.String();
+		should(Ti.Locale.currentCountry).eql('US');
+	});
+
+	it('currentLanguage', function () {
+		should(Ti.Locale.currentLanguage).be.a.String();
+		should(Ti.Locale.currentLanguage).eql('en');
+	});
+
+	it('currentLocale', function () {
+		should(Ti.Locale.currentLocale).be.a.String();
+		should(Ti.Locale.currentLocale).eql('en-US');
+	});
+
+	describe('#setLanguage(String)', function () {
+		it('is a Function', function () {
+			should(Ti.Locale.setLanguage).be.a.Function();
+		});
+
+		it('changes currentLanguage', function () {
+			Ti.Locale.setLanguage('fr');
+			should(Ti.Locale.currentLanguage).eql('fr');
+		});
+
+		// FIXME Get working on iOS, setLangauge doesn't seem to affect currentLocale
+		it.iosBroken('changes currentLocale', function () {
+			Ti.Locale.setLanguage('en-GB');
+			should(Ti.Locale.currentLocale).eql('en-GB'); // iOS returns 'en-US'
+			Ti.Locale.setLanguage('fr');
+			should(Ti.Locale.currentLocale).eql('fr');
+		});
+
+		// TODO test if it changes the currentCountry?
+	});
+
+	describe('#getString()', function () {
+		it('is a Function', function () {
+			should(Ti.Locale.getString).be.a.Function();
+		});
+
+		beforeEach(function () {
+			Ti.Locale.setLanguage('en-US');
+		});
+
+		it('returns stored value for found key', function () {
+			should(Ti.Locale.getString('this_is_my_key')).eql('this is my value');
+			should(L('this_is_my_key')).eql('this is my value');
+		});
+
+		it('returns key if not found and no default specified', function () {
+			should(Ti.Locale.getString('this_should_not_be_found')).eql('this_should_not_be_found');
+			should(L('this_should_not_be_found')).eql('this_should_not_be_found');
```

Comment:
<!--
  1 failure:  tests/Resources/t...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: tests/Resources/ti.locale.test.js;
  Line: 132;
-->
- :no_entry_sign: tests/Resources/ti.locale.test.js line 132 – 'L' is not defined. (no-undef)